src/DaPigGuy/PiggyCustomEnchants/enchants/weapons/bows/GrapplingEnchant.php

```diff
@@ -49,7 +49,7 @@ public function react(Player $player, Item $item, Inventory $inventory, int $slo
     {
         if ($event instanceof EntityDamageByChildEntityEvent) {
             $projectile = $event->getChild();
-            $task = new ClosureTask(function (int $currentTick) use ($event, $projectile) : void {
+            $task = new ClosureTask(function () use ($event, $projectile) : void {
```

Comment:
There shouldn't be a space before the colon (``use ($event, $projectile) : void {`` -> ``use ($event, $projectile): void {)``)